méli-mélo/2021-05-conjunction/index.html

```diff
@@ -100,10 +102,79 @@ <h4>Always stacked (mobile first, no side-by-side)</h4>
 	&lt;li class=&quot;cnjnctn-col&quot;&gt;
 		This is content for part B
 	&lt;/li&gt;
-&lt;/li&gt;</code>
+&lt;/li&gt;
+&lt;/ul&gt;</code>
   </pre>
   </details>
 
+<h3>No border on always stacked <span class="label label-info">New</span></h3>
+<p>This can be used when a border or design (e.g. zebra striping) has already defined the border of the content.</p>
+<h4>Always stacked (mobile first, no side-by-side)</h4>
+<div class="panel panel-default">
+    <header class="panel-heading">
+      <h5 class="panel-title">Heading</h5>
+    </header>
+    <div class="panel-body">
+  <ul class="cnjnctn-type-or brdr-0">
+<li class="cnjnctn-col">This is content for option A </li>
+    <li class="cnjnctn-col">This is content for option B</li>
+          </ul>
+	</div>
+</div>
```

Comment:
This structure need to be a sectioning element in order to have the "details/summary" content for the source code associated with the heading 4

Reply:
For example, wrapping both panel with a section element like:

```suggestion
<section>
<div class="panel panel-default">
    <header class="panel-heading">
      <h5 class="panel-title">Heading</h5>
    </header>
    <div class="panel-body">
  <ul class="cnjnctn-type-or brdr-0">
<li class="cnjnctn-col">This is content for option A </li>
    <li class="cnjnctn-col">This is content for option B</li>
          </ul>
	</div>
</div>
</section>
```

That way, the details/summary named "Code" will be associated with the title "Always stacked (mobile first, no side-by-side)" and not with the h5 named "Heading".